helm/elixier/templates/jupyterhub/statefulset.yaml

```diff
@@ -58,6 +58,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: K8S_NAMESPACE
```

Comment:
Can you remove this .. see if stuff still work?

Reply:
Removed commit and its jupyterhub & spark works